### PR TITLE
Fix patient category selection

### LINKED ISSUE #3729: Patient Category Filters for "Comfort Care" and "Slightly Abnormal" not working.
**To Reproduce**

1. Go to Patient Filters
2. Set category to Comfort Care / Slightly Abnormal
3. Apply it

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #3729 

- Fixes the category value to match with the backend, this resolves the error where the category value is an invalid choice.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Comment by rithviknishad** [comment]
@Ashesh3 three things:

- [ ] Consultation updates for "Slightly Abnormal" and "Comfort Care" not working.
- [ ] New log updates page not working.
- [ ] Edit log updates not working.

https://user-images.githubusercontent.com/25143503/195044508-fed8ab5b-886e-4166-8c31-a1ec4d3a1852.mp4

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Changes made in `PATIENT_CATEGORIES` constants need to be updated on log updates page, and also fix the consultation update not working for "Comfort Care" and "Slightly Abnormal"

**Comment by Ashesh3** [comment]
> @Ashesh3 three things:
> 
> * [x]  Consultation updates for "Slightly Abnormal" and "Comfort Care" not working.
> * [x]  New log updates page not working.
> * [x]  Edit log updates not working.
> 
>  2022-10-11.14-16-28.mp4

Should be fixed now.

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -264,7 +264,9 @@ export const ConsultationForm = (props: any) => {
               !!res.data.symptoms.filter((i: number) => i === 9).length,
             admitted: res.data.admitted ? String(res.data.admitted) : "false",
             admitted_to: res.data.admitted_to ? res.data.admitted_to : "",
-            category: res.data.category || "Comfort Care",
+            category: res.data.category
+              ? PATIENT_CATEGORIES.find((i) => i.text === res.data.category)?.id
```
Comment (khavinshankar) [inline src/Components/Facility/ConsultationForm.tsx]:
```suggestion
              ? PATIENT_CATEGORIES.find((i) => i.text === res.data.category)?.id || "Comfort"
```

File: src/Components/Patient/DailyRounds.tsx [hunk 2]
```diff
@@ -122,6 +122,11 @@ export const DailyRounds = (props: any) => {
         if (res && res.data) {
           const data = {
             ...res.data,
+            patient_category: res.data.patient_category
+              ? PATIENT_CATEGORIES.find(
+                  (i) => i.text === res.data.patient_category
+                )?.id
```
Comment (khavinshankar) [inline src/Components/Patient/DailyRounds.tsx]:
```suggestion
                )?.id || "Comfort"
```

File: src/Components/Patient/DailyRounds.tsx [hunk 3]
```diff
@@ -171,6 +176,11 @@ export const DailyRounds = (props: any) => {
           type: "set_form",
           form: {
             ...state.form,
+            patient_category: res.data.patient_category
+              ? PATIENT_CATEGORIES.find(
+                  (i) => i.text === res.data.patient_category
+                )?.id
```
Comment (khavinshankar) [inline src/Components/Patient/DailyRounds.tsx]:
```suggestion
                )?.id || "Comfort"
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**